app/build.gradle

```diff
@@ -44,6 +44,9 @@ dependencies {
     implementation "com.github.devlight.navigationtabstrip:navigationtabstrip:$rootProject.navTabStripVersion"
     implementation "com.afollestad.material-dialogs:core:$rootProject.materialDialogsVersion"
     implementation "com.jakewharton:butterknife:$rootProject.butterKnifeVersion"
+
+    implementation 'com.github.BeppiMenozzi:Knob:1.9.0'
```

Comment:
OK @abhinavraj23 now comes reviews! :D Add `1.9.0` to the gradle config file just like the other libraries.